### PR TITLE
QOLOE-22 Customising gutter value

### PR DESCRIPTION
**Pull request by ghazal-salehi**

### CODE REVIEW

**Review by stvp-qld — COMMENTED** [review]
.

**Review by duttonw — APPROVED** [review]

**Review by stvp-qld — CHANGES_REQUESTED** [review]
Can we check/confirm the 40px footer requirement please. The left/right rail of footer should be consistent with header and body content.

**Comment by ghazal-salehi** [comment]
The last change aligns the footer with qld-content. Example page: https://uat.forgov.qld.gov.au/user
Content page standard paint layout: https://oss-uat.clients.squiz.net/_admin/?screen=contents&assetid=471738

**Review by MitchAnderson93 — DISMISSED** [review]
Looks good

**Review by duttonw — APPROVED** [review]

**Review by duttonw — APPROVED** [review]

**Comment by duttonw** [comment]
Hi @ghazal-salehi ,

I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.

Regards,

@duttonw

**Comment by ghazal-salehi** [comment]
> Hi @ghazal-salehi ,
> 
> I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.
> 
> Regards,
> 
> @duttonw

The files changed are as what expected. Merge was still blocked. I did re-request just now.

**Review by duttonw — DISMISSED** [review]

File: src/components/bs5/footer/footer.scss
Comment (duttonw) [inline src/components/bs5/footer/footer.scss:21]: this should be changed to (--#{$prefix} per the others

File: src/components/common/layout/content.scss
Comment (duttonw) [inline src/components/common/layout/content.scss:6]: this should be changed to (--#{$prefix} instead of --qld 
so that prefix overrides are happy.

**Comment by duttonw** [comment]
> > Hi @ghazal-salehi ,
> > I've fixed the merge conflict and rebased your work to remove the noise. Can you please re-review your code changes and merge if its still correct.
> > Regards,
> > @duttonw
> 
> The files changed are as what expected. Merge was still blocked. I did re-request just now.

This was due to a 'request changes' pr review, I've dismissed that review, but there is extra tweaks work that needs to be done to ensure this is not flakey.

**Comment by ghazal-salehi** [comment]
The merge is blocked. Do you mind taking a look?

**Comment by duttonw** [comment]
> The merge is blocked. Do you mind taking a look?

Hi @jortronQGov, @stvp-qld, @re-becca,

Please PR review this change and approve.

Regards,

@duttonw

**Review by duttonw — COMMENTED** [review]
looks good, awaiting Online Experience team to provide approvals.

**Review by SenChung — APPROVED** [review]
Looks good.

**Review by elvishu — APPROVED** [review]
Looks good, Ghazal.

**Pull request merged**
